libraries/AP_OpticalFlow/AP_OpticalFlow_Calibrator.cpp

```diff
@@ -310,7 +310,12 @@ void AP_OpticalFlow_Calibrator::log_sample(uint8_t axis, uint8_t sample_num, flo
     // @Field: BRate: Body rate
     // @Field: LPred: Los pred
 
-    AP::logger().Write("OFCA", "TimeUS,Axis,Num,FRate,BRate,LPred", "QBBfff",
+    AP::logger().Write(
+        "OFCA",
+        "TimeUS,Axis,Num,FRate,BRate,LPred",
+        "s#-???",
```

Comment:
FRate, BRate and LPred are all rad/sec so these can be changed from "?" to "E".

Reply:
Done, thanks